### DISCUSSION TITLE
How can I use markdown in EditorContent?

### DISCUSSION THREAD

**Discussion by shoNagai** [ueberdosis/tiptap#2485] [General Questions]
I am using tiptap to create a simple editor. I am trying to use markdown notation, but it is not reflected in the editor.
The following is a gif image for verification, but even with a dash (-), it is not treated as an Unordered List.
![a8f0626c423d0459510d4938eabdd4a5](https://user-images.githubusercontent.com/22911189/152632600-8921c667-f285-466d-a9a0-eb4b81c63471.gif)
If you add a number sign (#) as an input character, it is not treated as a heading.

The repository is here.（Next.js + Typescript）
https://github.com/shoNagai/use-tiptap

I would appreciate your advice.

**Comment by hanspagel** [comment]
Looks like you need to add some CSS to make headings look different.

**Reply by shoNagai** [reply]
Thank you very much. I forgot about it.

Do you know of CSS examples that deal with tiptap editor in a similar configuration?
